gis/school_safety_zones/schools.py

```diff
@@ -231,6 +236,8 @@ def pull_from_sheet(con, service, year, spreadsheet, *args):
             cur.execute(truncate)  
             execute_values(cur, query, rows)
     LOGGER.info('Table %s is done', table)
+    if any_error:
+        raise AirflowFailException
```

Comment:
Is it possible to pass text to this? Such that the log line would be something like

```
[2022-11-28 13:56:12,548] {taskinstance.py:1150} ERROR - invalid rows pulled. See errors documented above.
```

Reply:
@tahaislam do you think there should be something logged before the `AirflowFailException` would appear?

Reply:
This's already resolved in this [commit](https://github.com/CityofToronto/bdit_data-sources/commit/0fb49c656f1cc1cba17930e138b897fd7d6319d7), but unfortunately, it's not showing here in this PR
https://github.com/CityofToronto/bdit_data-sources/blob/4eca6882859696f2794a5e5e0ab39ef029ee2618/gis/school_safety_zones/schools.py#L246

There're more commits in this branch that are not showing in this PR. And I don't know why this's happenning